### PR TITLE
Clean commented holymelon code

### PR DESCRIPTION
**Pull request by coiax**
Some commented code left in, commented code is bad, remove commented
code.

### CODE REVIEW

**Comment by RaveRadbury** [comment]
Wait are you removing the functionality here?

**Comment by interestingusernam3** [comment]
This was commented out in #55040 (newfood). It was probably an unintended change.

**Comment by RaveRadbury** [comment]
Should probably restore this rather than excising it if it was commented out during a _refactor_

**Comment by LemonInTheDark** [comment]
Restore it if you can, this looks like an accident to me

**Comment by Rohesie** [comment]
Was this intended, @Qustinnus ?

**Comment by Qustinnus** [comment]
No, I don't remember why i left it commented but it was definitely not intended.

**Comment by Rohesie** [comment]
> 
> 
> No, I don't remember why i left it commented but it was definitely not intended.

Was it intended to be removed or to be uncommented?

**Comment by Qustinnus** [comment]
uncommented

File: code/modules/hydroponics/grown/melon.dm [hunk 1]
```diff
@@ -82,23 +82,6 @@
...
-	if(!holy_person.mind?.holy_role || HAS_TRAIT(holy_person, TRAIT_AGEUSIA))
-		return
-	to_chat(holy_person,"<span class='notice'>Truly, a piece of heaven!</span>")
-	M.adjust_disgust(-5 + -2.5 * fraction)
-	SEND_SIGNAL(holy_person, COMSIG_ADD_MOOD_EVENT, "Divine_chew", /datum/mood_event/holy_consumption)
-	last_check_time = world.time
-
-
-*/
-
```
Comment (Rohesie) [inline code/modules/hydroponics/grown/melon.dm:101]: Do please uncomment this code instead of removing, as per the original author's intent.

**Pull request closed without merging**